src/prognostic_equations/vertical_diffusion_boundary_layer.jl

```diff
@@ -77,23 +129,87 @@ function vertical_diffusion_boundary_layer_tendency!(
 )
     ᶜρ = Y.c.ρ
     FT = Spaces.undertype(axes(ᶜρ))
-    (; ᶜp, ᶜspecific, sfc_conditions) = p.precomputed # assume ᶜts and ᶜp have been updated
-    (; C_E) = p.atmos.vert_diff
+    (; params) = p
+    thermo_params = CAP.thermodynamics_params(params)
+    (; ᶜts, ᶜp, ᶜspecific, sfc_conditions) = p.precomputed # assume ᶜts and ᶜp have been updated
+
+    κ = CAP.von_karman_const(params)
+    grav = CAP.grav(params)
+    
+    # TODO : Move to ClimaParameters
+    z₀ = FT(3.21e-5)
+    f_b = FT(0.1)
+    Ri_c = FT(1)
 
     ᶠgradᵥ = Operators.GradientC2F() # apply BCs to ᶜdivᵥ, which wraps ᶠgradᵥ
 
+    ᶠρK_E = p.scratch.ᶠtemp_scalar
+    θ_v = p.scratch.ᶜtemp_scalar
+    Ri = p.scratch.ᶜtemp_scalar_2
+    dz_local = p.scratch.ᶜtemp_scalar_3
+    θ_v_sfc = p.scratch.ᶠtemp_field_level
+    Ri_a = p.scratch.temp_field_level_2
```

Comment:
```suggestion
    Ri_a = p.scratch.temp_field_level
```
`temp_field_level` hasn't been used yet?